### ISSUE TITLE
mfa widget - "trouble scanning?" text is hard coded and cannot be translated

### ISSUE THREAD

**Issue by tomyam1** [auth0/lock#1951]
This is a small issue with the MFA widget.
I assume the same team maintains lock and it and since you don't host it publicly in github this is probably the best way to get you this message.

The "Trouble scanning?" label is hard coded and cannot be translated using the `languageDictionary` as proposed in https://community.auth0.com/t/replace-google-authenticator-title-with-okta/34875/2 and others sources.

Source: https://cdn.auth0.com/js/mfa-widget/mfa-widget-1.8.min.js

### Additional information, if any

![image](https://user-images.githubusercontent.com/3265649/102712877-d89f1d00-42cc-11eb-87b9-1bd4e78c4be5.png)

![image](https://user-images.githubusercontent.com/3265649/102712848-a2fa3400-42cc-11eb-83f3-006f3faa0d1c.png)

**Comment by stevehobbsdev** [comment]
Thanks for raising @tomyam1. Unfortunately no, it is not the same team but I have tracked down the owners and raised this internally with them. If and when I hear of a fix, I will try and report back here for you.

Closing for now as it's not an issue pertinent to this repo, but feel absolutely free to continue the discussion and like I said, I will report back if I hear of any progress.

**Comment by stevehobbsdev** [comment]
@tomyam1 I'm informed by the team that worked on this that this change has now been released in version 1.8 of the widget, and you should be able to customise the text as follows:

```js
totpEnrollment: {
  codePlaceholder: 'Enter your passcode here',
  headerText: 'Scan this QR code with Google Authenticator (or similar) app:',
  troubleScanning: 'Trouble Scanning ?'
},
```

Hope that helps!

**Comment by tomyam1** [comment]
Fixed now.
Thanks @stevehobbsdev!
